README.md

```diff
@@ -1,30 +1,5 @@
-# OhiouAcmWebsite
+# OHIO ACM
 
 This project acts as an information site for Ohio University's ACM Chapter.  
-The site was created using Angular and Firebase. Firebase is used for user login/authentication, and event/member data persistence.
 
 The admin account is ohiouacmwebsite@gmail.com
```

Comment:
should this be public?

Reply:
what are you talking about here

Reply:
This is already in the codebase. Are you requesting a force push with these erased? I'm not sure that'
s necessary